R/function_options.R

```diff
@@ -49,7 +49,7 @@ convert_function_options <- function(options, suppress_warnings) {
   } else if (!isTRUE(suppress_warnings) && length(unknown_option_names) > 0) {
     cli::cli_warn(c(
       "Found unknown {.code extendr} function option{?s}: {.val {unknown_option_names}}.",
-      inf_dev_extendr_used() # nolint: object_usage_linter
+      "i" = "Are you using a development version of {.code extendr}?"
```

Comment:
Don't need to quote these names.

Reply:
I opted for quoting all names passed to `cli`. See [cli_bullets()](https://cli.r-lib.org/reference/cli_bullets.html#details) examples.

I think the reason why the examples use quoting is because if you want to have `•` or `!` they need to be quoted. It can be odd for examples that might use `i = ` and following by `"!" = ` such as here https://github.com/extendr/rextendr/blob/afe61e61061d87bf41456ca91fc0c0ce2b5ff75b/R/eval.R#L88


```r
cli::cli_abort(
        c(
          "The Rust code fragment is no longer available for execution.",
            "i" = "Code fragment can only be executed once.",
            "!" = "Make sure you are not re-using an outdated fragment."
          )
      )
```

If you prefer to have this unquoted, please confirm and I will unquote all instances of `"i" = `